README.md

```diff
@@ -12,34 +13,33 @@ it only has two expressions.
 
 As an example, if your data looks like the following:
 
-```
+```javascript
 const json = {
   data: {
     Data: {
-      items: [1, 3, 5, 7]
+      items: [1, 3, 5, 7],
     },
-  }
+  },
 };
-
 ```
 
 You can select the `items` array as follows:
 
-```
-import jsl from 'json-selector-lang';
-
-const itemsSelector = jsl.compile('.data.Data.items');
-
-itemsSelector
-  .eval(json)
-  .then(val => {
-    // val === [1, 3, 5, 7]
-  })
-  .catch((e) => {
-    // interpreter error
-  });
+```javascript
+import { JSL, JSLEvaluator } from 'json-selector-lang';
+
+try {
+  const itemsSelector = JSL.compile('.data.Data.items');
+
+  const evaluator = new JSLEvaluator();
+  const items = evaluator.evaluate(data, itemsSelector);
```

Comment:
i think this looks a bit awkward, I have a suggestion though. Let me know if you are ok with it

**Suggestion:** 
we can add the evaluator in JSL like this
```javascript
class JSL {
  static compile(input: string): Program {}
  static evaluate(data: any, node: ASTNode): {
    const evaluator = new JSLEvaluator();
    
    return evaluator.evaluate(data, node);
  }
}
```
and we can use it like this
```javascript
import { JSL } from 'json-selector-lang'

try {
  const itemsSelector = JSL.compile('.data.Data.items');
  const items = JSL.evaluate(data, itemsSelector);
} catch(e) {
  // catch the errors here
}
```

Reply:
Yes lets make it as lowered case, i think it should be something like this
```javascript
import { jsl } from 'json-selector-lang'
```